### DISCUSSION TITLE
WebSocket seems to be reopened instead of being opened new if you parralelize functions

### DISCUSSION THREAD

**Discussion by sayore** [cc-tweaked/CC-Tweaked#1227] [General]
Main starts a wait for Any

FunctionA Starts a Script -> Starts another seperate WebSocket, using another Protocol internally, same WS Address as Function B uses
FunctionB watches a WebSocket

When i now wanna close the WebSocket in Function A, it seems like Function B loses it's connection as well, but this is closed by CC not the Server(so it's not like the Socket gets mixed up on WS End)
So either FunctionA gets a clone from Function B
Or FunctionA reopens the WebSocket, and leaves the first WebSocket in the Limbo?

**Comment by Lupus590** [comment]
Can you post your code?
